sencha-workspace/SlateAdmin/app/controller/People.js

```diff
@@ -24,6 +24,7 @@ Ext.define('SlateAdmin.controller.People', {
     routes: {
         'people': 'showPeople',
         'people/all': 'showResults',
+        'people/create': 'createPerson',
```

Comment:
this should be called `showCreatePerson` as all other route handlers and its equivalent in the sections controller is named. The body of a route handler should be positioned in the controller at the top section in the same order among other route handlers as this config is positioned (i.e. put it under `showResults`)